src/gui/widgets/TabWidget.cpp

```diff
@@ -48,6 +48,7 @@ TabWidget::TabWidget(const QString & caption, QWidget * parent, bool usePixmap,
 	m_tabText( 0, 0, 0 ),
 	m_tabTitleText( 0, 0, 0 ),
 	m_tabSelected( 0, 0, 0 ),
+	m_tabTextSelected( 0, 0, 0 ),
```

Comment:
```suggestion
	m_tabTextSelected(0, 0, 0),
```

Reply:
Spaces fixed with commit 0fa18d405e7 where I have fixed style issues for all of `TabWidget`.